Themis/Views/Assessment/CorrectionSidebar/FeedbackListView.swift

```diff
@@ -17,6 +17,8 @@ struct FeedbackListView: View {
     var pId: Int?
     var templatePId: Int?
     
+    let exercise: ExerciseOfSubmission?
```

Comment:
This component also only needs to get the `gradingCriteria` instead of the whole exercise.

Reply:
Should be in this commit, which I have pushed earlier: https://github.com/ls1intum/Themis/pull/75/commits/cd4c5c56a4c7734b7b40c6cd461cdf7650748d5f